msg/tools/uorb_rtps_message_ids.yaml

```diff
@@ -177,8 +177,10 @@ rtps:
     id: 76
   - msg: test_motor
     id: 77
-  - msg: timesync_status
```

Comment:
Is there a reason you flipped these?

Reply:
Not really. Just seemed more consistent to me.